ex2/api/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "api",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js",
+    "server": "nodemon index.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
```

Comment:
I'm sure it would be better for experience not to use ready-made frameworks to create a server, because this is part of learning, to figure out how the web server is arranged and works inside, at least in the most minimal form. 
The Express developers have received the experience for organizing routing, servicing CORS, and correct division into layers instead of you)
So let's try reworking it to solution without Express, but based on your solution from Exercise 1, as it requested into task specification.